src/pages/Wallet/WalletOverview/components/WalletInsights/WalletInsights.tsx

```diff
@@ -33,9 +45,53 @@ const WalletInsights = ({ balances }: WalletInsightsProps): JSX.Element => {
       new Big(0)
     );
 
+  const totalLendPositions =
+    lendPositions &&
+    Object.values(lendPositions).reduce(
+      (total, balance) =>
+        total.add(
+          convertMonetaryAmountToValueInUSD(
+            balance.amount,
+            getTokenPrice(prices, balance.amount.currency.ticker)?.usd
+          ) || 0
+        ),
+      new Big(0)
+    );
+
+  const totalBorrowPositions =
+    borrowPositions &&
+    Object.values(borrowPositions).reduce(
+      (total, balance) =>
+        total.add(
+          convertMonetaryAmountToValueInUSD(
+            balance.amount,
+            getTokenPrice(prices, balance.amount.currency.ticker)?.usd
+          ) || 0
+        ),
+      new Big(0)
+    );
+
+  const totalAccountLiquidityPools =
+    accountLiquidityPools &&
+    Object.values(
+      accountLiquidityPools.map(({ data, amount: accountLPTokenAmount }) => {
+        const { pooledCurrencies, totalSupply } = data;
+        const totalLiquidityUSD = calculateTotalLiquidityUSD(pooledCurrencies, prices);
+
+        return accountLPTokenAmount
+          ? calculateAccountLiquidityUSD(accountLPTokenAmount, totalLiquidityUSD, totalSupply)
+          : 0;
+      })
+    ).reduce((total, accountLPTokenAmount) => total.add(accountLPTokenAmount), new Big(0));
```

Comment:
Similarly to previous values, I think it would be better to move this computation to `useGetAccountPools` hook,